vega-lite-schema.json

```diff
@@ -2563,19 +2553,9 @@
                     "type": "number"
                 },
                 "rangeStep": {
-                    "anyOf": [
-                        {
-                            "enum": [
-                                "fit"
-                            ],
-                            "type": "string"
-                        },
-                        {
-                            "type": "number"
-                        }
-                    ],
                     "description": "Default range step for (1) `y` ordinal scale,\nand (2) `x` ordinal scale when the mark is not `text`.",
-                    "minimum": 0
+                    "minimum": 0,
+                    "type": "number"
```

Comment:
@domoritz  looks like there is a bug in `typescript-json-schema` ?   

This is from `number | null` and thus should include `null` in the schema.